drone_wrapper build failing due to drone_tello_class

Build failure log output: https://build.ros.org/job/Nbin_ufhf_uFhf__drone_wrapper__ubuntu_focal_armhf__binary/18/console

Main issue :
```bash
23:16:26 CMake Error at cmake_install.cmake:60 (file):
23:16:26   file INSTALL cannot find
23:16:26   "/tmp/binarydeb/ros-noetic-drone-wrapper-1.4.1/src/drone_tello_class.py":
23:16:26   No such file or directory.
23:16:26 
```

--- a/drone_assets/package.xml
+++ b/drone_assets/package.xml
@@ -10,7 +10,7 @@
   <license>MIT</license>
 
   <buildtool_depend>catkin</buildtool_depend>
-
+  <depend>gazebo_ros</depend>
 
   <!-- The export tag contains other, unspecified, tags -->
   <export>

